src/kernel/mreader.ml

```diff
@@ -66,16 +66,23 @@ let instantiate_reader tr spec config source = match spec with
   | name :: args ->
     let reader = lazy (Mreader_extend.start tr name args config source) in
     (reader, (fun () ->
-       if Lazy.is_val reader then
-         match Lazy.force reader with
-         | None -> ()
-         | Some reader -> Mreader_extend.stop tr reader))
+         if Lazy.is_val reader then
+           match Lazy.force reader with
+           | None -> ()
+           | Some reader -> Mreader_extend.stop tr reader))
```

Comment:
That diff shouldn't be here.

Reply:
ocp-indent failure I think - will try to revert it